docs/sdk/access-control/encryption.md

```diff
@@ -95,7 +95,7 @@ const accessControlConditions = [
 ];
 ```
 
-#### AuthSig
+#### Passing AuthSig
```

Comment:
Minor comment, but can we add 'an' here? So full sentence being 'Passing an AuthSig'